packages/wrangler/src/user.tsx

```diff
@@ -365,6 +372,13 @@ function throwIfNotInitialised() {
 
 export function getAPIToken(): string | undefined {
   throwIfNotInitialised();
+  if (LocalState.api_token) {
+    console.warn(
+      "It looks like you have used Wrangler 1's `config` command to login with an API token.\n" +
```

Comment:
tempted to just ignore the field and proceed as if the user isn't logged in.

Reply:
That is what we do at the moment, and it was causing confusion. See https://github.com/cloudflare/wrangler2/issues/526

Hence this PR.